lib/src/git.dart

```diff
@@ -57,7 +57,10 @@ Future<List<String>> run(List<String> args,
   log.muteProgress();
   try {
     var result = await runProcess(command, args,
-        workingDir: workingDir, environment: environment);
+        /// Set git language to English to pass test/global/activate/missing_git_repo_test.dart
```

Comment:
[nit] If we keep this comment it should use 2 slashes.

Since we have a test for this I'd recommend skipping the comment - the test will prevent us from thinking we can remove it.

Reply:
You are right, three slashes is for public API documentation. I guess it is clear what this line of code is doing.